lightapi/__init__.py

```
@@ -1,6 +1,6 @@
 from .auth import JWTAuthentication
 from .cache import RedisCache
-from .core import LightApi, Middleware, Response
+from .core import LightApi, Middleware, Response, CORSMiddleware, AuthenticationMiddleware
```

Ruff (0.11.9)

`.core.CORSMiddleware` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)